src/lib/utils/drupalClient.ts

```diff
@@ -16,8 +17,14 @@ export const fetcher = async (input: RequestInfo, init?: RequestInit) => {
     ...init,
   }
 
-  return crossFetch(input, {
-    ...options,
+  // Wrap fetching in p-retry for resilience.
```

Comment:
Did this because if a single API call failed for whatever reason, the entire export would fail. This just retries a few times.